src/format/context/common.rs

```diff
@@ -31,6 +31,12 @@ impl Context {
 	pub unsafe fn destructor(&self) -> Rc<Destructor> {
 		Rc::clone(&self.dtor)
 	}
+
+	// Create a new unsafe reference to 'Context' with a new lifetime,
+	// to pass to a 'wrap()' function.
```

Comment:
Comment here is my understanding of what this is for...

I am wondering, given a `Context` is a libavcodec pointer and a reference counted destructor, if it would be better to implement `std::clone` and store clones of `Context` in Streams instead of storing an unsafe reference.

Is that too slow or something? Sorry if this is way off-base due to me not understanding libavcodec.